### DISCUSSION TITLE
In Japan, ozoni soup is traditionally served on the morning of what day?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#623] [Galaxy Trivia] [answered]
**Bonus:** List main ingredients.

**Comment by MohammedNayeem9** [comment, marked as answer]
**Ozoni**  soup🍛  is traditionally served on the morning of **New Year's Day** in Japan 🇯🇵.
**List of main ingredients**
1. Dashi ( broth made from kombu forms a miso soup🍲). 
2. Mochi (rice cake 🍚). 
3. Vegetables such as
- carrots 🥕
- daikon radish
- shiitake mushrooms 🍄
- Spinach 🥬 or other leafy greens 
4. Chicken, fish, or tofu. 
5. Soy sauce or salt 🧂. 
6. Yuzu citrus 🍊 peel or other herbs.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/MohammedNayeem9?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉
